apps/dashboard/src/components/ipfs-upload/dropzone.tsx

```diff
@@ -492,6 +509,19 @@ const FileUpload: React.FC<FileUploadProps> = ({ files, updateFiles }) => {
                 colorScheme="green"
                 leftIcon={<Icon as={FiUploadCloud} />}
                 onClick={() => {
+                  // If one of the files belong to the "block list", do not allow to proceed
+                  const invalidFile = files.find((file) =>
+                    UNACCEPTED_FILE_TYPES.includes(file.type),
+                  );
+                  if (invalidFile) {
+                    return toast({
+                      title: "Error",
+                      description: `${invalidFile.type} files are not accepted`,
+                      status: "error",
+                      isClosable: true,
+                      duration: 4000,
+                    });
+                  }
```

Comment:
you do not need this as it will be impossible to add invalid files in the first place
```suggestion
```